modules/ossm-deploy-application-workloads-in-each-mesh.adoc

```diff
@@ -0,0 +1,210 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploy-application-workloads-in-each-mesh_{context}"]
+= Deploy application workloads in each mesh
+:context: ossm-deploy-application-workloads-in-each-mesh
+
+This procedure explains how to configure a cluster to host two control planes, each managing independent service mesh data planes. The configuration involves creating separate `{istio}` resources with unique names in independent {istio} system namespaces. Eah `{istio}` resource will use a unique `revision` name, which identifies the control planes, workloads, or namespaces associated with it. These `revision` names, applied using injection or `istio.io/rev` labels, enable sidecar injection while also indicating the corresponding control plane.
+
+Each `{istio}` resource must also configure `discoverySelectors` to specify which namespaces the {istio} control plane observes. Only namespaces with labels that match the configured `discoverySelectors` can join the mesh.
+
+When adding an additional {istio} control plane to a cluster with an existing control plane, ensure that the existing `{istio}` instance has `discoverySelectors` configured to avoid overlapping with the new control plane. Run the following command to check for existing `{istio}` instances.
+
+This procedure may be extended to support more than two control planes. The primary limitation on the number of service meshes in a single cluster is the availability of cluster resources.
+
+[NOTE]
+====
+There is only one `IstioCNI` resource shared by all control planes in the cluster, and it must be updated independently.
+====
+
+[source,terminal]
+----
+$ oc get istios
+----
+
+
+.Procedure
+
+. Create an application namespace called `app-ns-1`:
++
+[source,terminal]
+----
+$ oc create namespace app-ns-1
+----
+
+. Add the `usergroup=usergroup-1` label to ensure that the namespace is discovered by the first control plane and the `istio.io/rev=mesh-1` to enable sidecar injection in the namespace by default.
```

Comment:
Based on feedback from Marko, we'll remove the "usergroup" naming as it is not an Istio/ossm concept and could confuse. I also think this step should be split up for clarity as it is combining two things into one step. So I suggest this step is changed to:

```suggestion
. Add the `istio-discovery=mesh-1` label to ensure that the namespace is discovered by the first control plane:
```
The command becomes:
```
oc label namespace app-ns-1 usergroup=usergroup-1 
```
With a new step...
1. Add the `istio.io/rev=mesh-1` label to the namespace to enable sidecar injection by default while ensuring that pods in this namespace are mapped to the first control plane.
```
oc label namespace app-ns-1 istio.io/rev=mesh-1
```